bibliopixel/remote/server.py

```diff
@@ -5,36 +5,19 @@
 import os
 import logging
 from .. import log
+import sys
 
-# I want to keep this to supress the "Running on" message
-# But then it also swallows 404 erros and such in the console :(
-# werk_log = logging.getLogger('werkzeug')
-# werk_log.setLevel(logging.ERROR)
+if '--verbose' not in sys.argv:
```

Comment:
Yeah... exactly. I tried for an hour to make that work cleaner without a boatload of code changes and in the end just decided that this way was quicker and that in the future we would probably come up with a cleaner solution for passing global options through the system.